### PR TITLE
Demo

### PR DESCRIPTION
**Pull request by QuanWan89**

### CODE REVIEW

File: amplify/team-provider-info.json [hunk 1]
```diff
@@ -14,7 +14,13 @@
     },
     "categories": {
       "auth": {
+<<<<<<< HEAD
+        "undefined": {},
+        "cmgd": {},
+        "true": {}
+=======
         "cmgd": {}
+>>>>>>> main
```
Comment (tiandin) [inline amplify/team-provider-info.json:23]: Seems like there are some conflicts need to be resolved?

File: amplify/backend/api/cmgdapi/src/controllers/project.js [hunk 2]
```diff
@@ -47,6 +47,12 @@ module.exports = {
       }
     )
       .then((project) => res.status(201).send(project))
-      .catch((error) => res.status(400).send(error));
+      .catch((error) => {
+        if (error.name === "SequelizeUniqueConstraintError") {
+          res.status(400).json({ message: "projectId must be unique" });
+        } else {
+          res.status(400).send(error);
```
Comment (tiandin) [inline amplify/backend/api/cmgdapi/src/controllers/project.js:54]: Can we also get the error message from client side if the error is not ```SequelizeUniqueConstraintError```?

**Review by tiandin — COMMENTED** [review]
2 comments

**Review by tiandin — APPROVED** [review]
LGTM

File: amplify/backend/api/cmgdapi/src/controllers/project.js (hunk 2 above)
Comment (QuanWan89) [reply amplify/backend/api/cmgdapi/src/controllers/project.js]: 9df60d04d899ddf32f5bdfbb9108be6dc5e4a787

**Pull request merged**